WooCommerce/Classes/ViewRelated/ReusableViews/TextViewTableViewCell.swift

```diff
@@ -3,9 +3,12 @@ import Gridicons
 
 class TextViewTableViewCell: UITableViewCell {
 
+    @IBOutlet private weak var stackView: UIStackView!
```

Comment:
nit: seems like this `stackView` isn't used? maybe we don't need this IB outlet?